Makefile

```diff
@@ -144,9 +144,6 @@ tmp/$(SYSTEM)/default/shared/glew.o: src/glew.c include/GL/glew.h include/GL/wgl
 glew.pc: glew.pc.in
 	sed \
 		-e "s|@prefix@|$(GLEW_PREFIX)|g" \
-		-e "s|@libdir@|$(LIBDIR)|g" \
```

Comment:
Looks like LIBDIR varies.

```
$ grep LIBDIR config/*
config/Makefile.cygming:LIBDIR = /usr/lib/mingw
config/Makefile.cygming:LDFLAGS.EXTRA = -L$(LIBDIR)
config/Makefile.cygwin:LIBDIR = $(GLEW_DEST)/lib
config/Makefile.cygwin-x11:LIBDIR = $(GLEW_DEST)/lib
config/Makefile.flatpak-32:LIBDIR = $(GLEW_DEST)/lib32 
config/Makefile.linux:  LIBDIR = $(GLEW_DEST)/lib64
config/Makefile.linux:  LIBDIR = $(GLEW_DEST)/lib
config/Makefile.linux-32:LIBDIR = $(GLEW_DEST)/lib
config/Makefile.linux-clang:  LIBDIR = $(GLEW_DEST)/lib64
config/Makefile.linux-clang:  LIBDIR = $(GLEW_DEST)/lib
config/Makefile.msys:LIBDIR = $(GLEW_DEST)/bin
config/Makefile.nacl-32:LIBDIR =
config/Makefile.nacl-64:LIBDIR =
```

Same rationale for BINDIR and INCDIR, I expect.